### DISCUSSION TITLE
Does this octo.nvim work alongside Team Foundation Server (TFS)?

### DISCUSSION THREAD

**Discussion by galenseilis** [pwntester/octo.nvim#651] [Q&A]
Someone just recommended I check out this plugin. Looks very promising for enhancing my workflow.

For my personal stuff I use Github so I see the immediate value of this plugin. 

For work I also use TFS, and the local git is setup to see repos there as the remote.

Does this plugin work with TFS, provided  the dependencies listed in the README are installed, or is it strictly Github?

Thanks,

**Comment by galenseilis** [comment]
Or would I need some additional dependencies such as https://github.com/git-tfs/git-tfs ?

**Comment by williambdean** [comment]
I am less familiar with the tool TFS @galenseilis 

The commands are all centered around `gh` CLI and using the GraphQL and REST endpoints. `git` binary and various commands are also used directly
